docs/extras/guides/privacy/presidio_data_anonymization/multi_language.ipynb

```diff
@@ -190,21 +190,99 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "My name is Ronnie Ayala\n"
+      "My name is Brian Johnson\n"
      ]
     }
    ],
    "source": [
     "print(anonymizer.anonymize(\"My name is John\"))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Usage with other tools\n",
+    "\n",
+    "### Language detection\n",
```

Comment:
If we have it here, remove "automatic language detection" from future works - basically, you can delete future works section now XD

Reply:
done :+1: